src/encoded/static/scss/encoded/modules/_mouse_map.scss

```diff
@@ -1,20 +1,40 @@
 .mouse-map {
     .cls-1, .cls-bone1, .cls-limb1 {
-        fill: #e4ecf3;
+        fill: #b7c9d8;
 
         &.active {
             fill: #f7e1d4;
         }
     }
 
-    .cls-2, .cls-limb2 {
-        fill: #bbd0e2;
+    .cls-limb2 {
+        fill: #d8e2ea;
 
         &.active {
             fill: #ccbdb4;
         }
     }
 
+    .cls-mousebody {
+        stroke: #85a2b9;
+        fill: #d8e2ea;
+
+        &.active {
+            // fill: #ccbdb4;
```

Comment:
Why not remove this comment as it serves no purpose?

Reply:
I guess I'm a little torn on this one. I shouldn't be storing colors in comments, but I think there's a chance at some point we'll want to highlight the mouse body and this is the color for it.